src/style.css

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header__nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
Replace RGBA with HEX

Reply:
is it possible to use HEX for the box-shadow? I mean the opacity part. I was looking the syntax [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#syntax)

Reply:
Box shadows have offset-x | offset-y | blur-radius so add the blur radius and proper color to handle opacity

Reply:
got it, thank you